windows-agent/internal/config/config_test.go

```diff
@@ -391,7 +401,18 @@ func TestSetUserLandscapeConfig(t *testing.T) {
 			conf := config.New(ctx, dir)
 			setup(t, conf)
 
-			landscapeConfig := "LANDSCAPE CONFIG"
+			wantSource := config.SourceUser
+			switch tc.landscapeConfig {
+			case "":
+				tc.landscapeConfig = landscapeBaseConf
+				if tc.withHostagentUID {
+					tc.landscapeConfig += "\nhostagent_uid=discard_me\n"
```

Comment:
I prefer `new_and_discarded_hostagent_uid` in this case because this test case illustrates that the `hostagent_uid` is ignored if present in the user-supplied Landscape config (as that value must always come assigned from the Landscape server itself).